docs_user/modules/proc_deploying-backend-services.adoc

```diff
@@ -1,31 +1,24 @@
 [id="deploying-backend-services_{context}"]
 
-//:context: backend-services
-
-//kgilliga: The backend services deployment intro will be converted to an assembly. Most of the rest of the module will be converted to a procedure.
-//Check xref context.
-
 = Deploying backend services
 
-The following instructions create OpenStackControlPlane CR with basic
-backend services deployed, and all the OpenStack services disabled.
-This will be the foundation of the podified control plane.
+Create the `OpenStackControlPlane` custom resource (CR) with basic backend services deployed, and all the {osp_prev_long} ({OpenStackShort}) services disabled. This will be the foundation of the control plane.
 
 In subsequent steps, you import the original databases and then add
-podified OpenStack control plane services.
+{OpenStackShort} control plane services.
 
-== Prerequisites
+.Prerequisites
 
 * The cloud that you want to adopt is up and running, and it is on the
-OpenStack Wallaby release.
+{OpenStackShort} {osp_prev_ver} release.
```

Comment:
same here

Reply:
I think in this case we want the sentence to say "RHOSP 17.1", right? The customer's environment needs to be running RHOSP 17.1 before they begin deploying backend services.